parity/secretstore/server.rs

```diff
@@ -134,6 +134,11 @@ mod server {
 
 	const SECP_TEST_MESSAGE: H256 = H256([1_u8; 32]);
 
+	/// Version of the secret store database
+	const SECRET_STORE_DB_VERSION: &str = "4";
+	/// Version file name
+	const VERSION_FILE_NAME: &str = "db_version";
```

Comment:
```suggestion
	const SECRET_STORE_DB_VERSION_FILE_NAME: &str = "db_version";
```
We have a `VERSION_FILE_NAME` already for the main DB, so best stay explicit (but if you can think of something less absurdly long, that's fine).